### DISCUSSION TITLE
进入阅读模式后无法查看图片的解决方案

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1558] [使用技巧]
> 举例说明，如访问 https://cloud.tencent.com/developer/article/1120756 可以进入阅读模式，但无法显示图片的话，可以试试下面两个解决方法：

***

1. 把 `https://cloud.tencent.com/developer/article/*` 加入到 **[白名单](http://ksria.com/simpread/docs/#/FAQ?id=%e7%99%bd%e5%90%8d%e5%8d%95)**

2. 把 `https://cloud.tencent.com/developer/article/*` 加入到 **[延迟加载列表](http://ksria.com/simpread/docs/#/%E8%AF%8D%E6%B3%95%E5%88%86%E6%9E%90%E5%BC%95%E6%93%8E?id=%e5%bb%b6%e8%bf%9f%e5%8a%a0%e8%bd%bd)**

***

1. 阅读模式 → （右下角）选项 → 动作 → 设定 → 站点编辑器，打开后 **直接保存**

2. 把 `https://cloud.tencent.com/developer/article/*` 加入到 **[延迟加载列表](http://ksria.com/simpread/docs/#/%E8%AF%8D%E6%B3%95%E5%88%86%E6%9E%90%E5%BC%95%E6%93%8E?id=%e5%bb%b6%e8%bf%9f%e5%8a%a0%e8%bd%bd)**

***

> 备注

1. 这两种方式请注意添加顺序

2. 建议使用 **第一种方案**

3. 加入的 [URL 多种模式](http://ksria.com/simpread/docs/#/%E7%AB%99%E7%82%B9%E7%BC%96%E8%BE%91%E5%99%A8?id=%e5%af%b9%e5%ba%94%e5%ad%97%e6%ae%b5)，不一定费用使用通配符方案

**Comment by YangWaldon** [comment]
> > 举例说明，如访问 https://cloud.tencent.com/developer/article/1120756 可以进入阅读模式，但无法显示图片的话，可以试试下面两个解决方法：
> 
> 1. 把 `https://cloud.tencent.com/developer/article/*` 加入到 **[白名单](http://ksria.com/simpread/docs/#/FAQ?id=%e7%99%bd%e5%90%8d%e5%8d%95)**
> 2. 把 `https://cloud.tencent.com/developer/article/*` 加入到 **[延迟加载列表](http://ksria.com/simpread/docs/#/%E8%AF%8D%E6%B3%95%E5%88%86%E6%9E%90%E5%BC%95%E6%93%8E?id=%e5%bb%b6%e8%bf%9f%e5%8a%a0%e8%bd%bd)**
> 
> 1. 阅读模式 → （右下角）选项 → 动作 → 设定 → 站点编辑器，打开后 **直接保存**
> 2. 把 `https://cloud.tencent.com/developer/article/*` 加入到 **[延迟加载列表](http://ksria.com/simpread/docs/#/%E8%AF%8D%E6%B3%95%E5%88%86%E6%9E%90%E5%BC%95%E6%93%8E?id=%e5%bb%b6%e8%bf%9f%e5%8a%a0%e8%bd%bd)**
> 
> > 备注
> 
> 1. 这两种方式请注意添加顺序
> 2. 建议使用 **第一种方案**
> 3. 加入的 [URL 多种模式](http://ksria.com/simpread/docs/#/%E7%AB%99%E7%82%B9%E7%BC%96%E8%BE%91%E5%99%A8?id=%e5%af%b9%e5%ba%94%e5%ad%97%e6%ae%b5)，不一定费用使用通配符方案

![image](https://user-images.githubusercontent.com/49675185/115131510-406f3580-a02b-11eb-9aed-db849ea6cd5e.png)
进入之后无法查看图片，我的预加载机制本来就是关闭的，按照上述方法把当前页面添加到了白名单，之后再进阅读模式就变成这样了，另外，重新高亮区域进阅读模式打开站点编辑器，保存，刷新页面，彩云小译翻译，进阅读模式，还是无法正确识别网页，请问有解决办法嘛

**Reply by YangWaldon** [reply]
启用增强解析模式后解决了公式混乱的问题，图片也可以显示了（看来是加入白名单的方式起作用了），但是重新高亮的问题依然没有解决，请问有解决办法吗

**Reply by Kenshin** [reply]
请到 https://github.com/Kenshin/simpread/issues/2017 里面进行统一回复
